.takt/facets/instructions/ai-review.md

```
@@ -1,23 +1,7 @@
-AI生成コード特有の問題をレビューせよ。
+AI生成コード特有の問題をレビューしてください。
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)